client/js/navigation/RootStackNavigator.js

```diff
@@ -4,7 +4,7 @@ import {
   createStackNavigator
 } from "react-navigation";
 import { AppStack } from "./NavigationLayout";
-import SignInScreen from "../screens/SignIn";
+import SignInScreen from "../screens/LogIn";
```

Comment:
Naming convention is inconsistent here. Let's try to match file names to class names.

Reply:
Fixed folder names and where these are imported as well to match class names - thanks.